torch/csrc/lazy/core/trie.h

```diff
@@ -57,8 +61,9 @@ NodePtr LookupNodeFromTrieCache(Args&&... args) {
   for (auto it = successors.begin(); it != successors.end(); it++) {
     NodePtr ir_node = (*it)->ir_node;
     const T* concrete_node = NodeCast<T>(ir_node.get());
-    if (concrete_node && concrete_node->Equal(std::forward<Args>(args)...)) {
+    if (concrete_node && concrete_node->CanBeReused(std::forward<Args>(args)...)) {
       TORCH_LAZY_COUNTER("IrNodeReused::" + std::string(typeid(T).name()), 1);
```

Comment:
i do not know.  it would be nice to do this though, as it looks horrible to have those mangled names show up :)

https://stackoverflow.com/questions/281818/unmangling-the-result-of-stdtype-infoname
i didn't read carefully, it looks like there are some solutions but it might be tricky to get it working with all of our different compilers.